src/main/kotlin/Command/base/Processor.kt

```diff
@@ -0,0 +1,19 @@
+package Command.base
+
+import Command.commands.ChooseIngredientCommand
+import Command.commands.ChooseNoodlesCommand
+import Command.commands.ChooseSauceCommand
+import Command.commands.ShowPriceCommand
+
+class Processor(val cart: Cart) {
+
+  fun process(command : String) : Command {
```

Comment:
You are 👀 my implementation, you can see that I have implemented a queue of process